e3db/types/note.py

```diff
@@ -405,7 +389,7 @@ def is_expires(self) -> bool:
         expires : bool
             Truth value determining whether the note has expired
         """
-        return self.__expires
+        return self.__note_options.expires
```

Comment:
should there be some check that this value is less than the current time?

Reply:
I believe this is outside the scope of this subtask. I would like to keep this PR to just the note encrypt + sign. I suggest a follow up subtask to go through the Note object and add some tests and verify functionality (and add anything that's missing)